scripts/upgradeable/patch-apply.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+DIRNAME="$(dirname -- "${BASH_SOURCE[0]}")"
+PATCH="$DIRNAME/upgradeable.patch"
```

Comment:
I'm wondering if we should have multiple patch files:
```$DIRNAME/upgradeable-patches/*.patch```

The advantage is that we could more effectivelly segment the different things that are patched (instead of having a single difficult to read patch file). The downside is that patch generation is more complexe. Maybe that is something we can delay until we actually need it (if ever)

Reply:
Yeah the reason for a single patch is that `patch-save.sh` can be super simple.